examples/experimental/i3s-17-and-debug/helpers/build-minimap-data.js

```diff
@@ -0,0 +1,25 @@
+import {Vector3} from '@math.gl/core';
+import {Ellipsoid} from '@math.gl/geospatial';
+import {OrientedBoundingBox} from '@math.gl/culling';
+
+export function buildMinimapData(tiles) {
+  return tiles
+    .map(tile => {
+      if (!tile.selected || !tile.viewportIds.includes('main')) {
```

Comment:
Maybe start with calling `.filter` instead with this logic, rather than filtering at the end?

Reply:
Filtering should be done after mapping because in mapping we do `return null` under some conditions.